### DISCUSSION TITLE
Watching or following the tournament?

### DISCUSSION THREAD

**Discussion by goosehub** [SebLague/Chess-Challenge#358] [Ideas]
It is maybe too early to discuss these details, but I was wondering and hoping when the tournament arrives that there will be a way to either watch live, or get the results of the match, with PGNs if possible. It would be fun to watch the bots and exciting to see my bots performance during the tournament. If it's only going to be revealed in a video that's fine too of course, probably far easier for Seb, but I thought I'd float this idea just in case it hasn't already been considered.

**Comment by WhiteMouse1** [comment]
Community discord will hold community tournaments, all of which will be streamed.
They will definitely stream official one too.
